src/cplscheme/CouplingData.cpp

```diff
@@ -119,19 +119,16 @@ void CouplingData::storeIteration()
 
 const Eigen::VectorXd CouplingData::previousIteration() const
 {
-  PRECICE_ASSERT(!_previousTimeStepsStorage.stamples().empty());
```

Comment:
Why did you remove these assertions? Or was this an accident from the force-pushing here?